packages/node_modules/@webex/plugin-meetings/src/locus-Info/index.js

```diff
@@ -0,0 +1,238 @@
+import isEqual from 'lodash.isequal';
+
+/**
+ * @export
+ * @class LocusInfo
+*/
+export default class LocusInfo {
+  constructor() {
+    this.setup({});
+  }
+
+  setup(locus) {
+    this.controls = locus.controls || null;
+    this.conversationUrl = locus.conversationUrl || null;
+    this.created = locus.created || null;
+    this.fullState = locus.fullState || null;
+    this.host = locus.host || null;
+    this.info = locus.info || null;
+    this.mediaShares = locus.mediaShares || null;
+    this.participantsUrl = locus.participantsUrl || null;
+    this.replaces = locus.replaces || null;
+    this.self = locus.self || null;
+    this.url = locus.url || null;
+    this.aclUrl = locus.aclUrl || null;
+    this.baseSequence = locus.baseSequence || null;
+    this.sequence = locus.sequence || null;
+    this.membership = locus.membership || null;
+    this.identities = locus.identities || null;
+    this.participants = locus.participants || null;
+  }
+
+  onFullLocus(locus) {
+    this.updateLocusInfo(locus);
+    this.updateParticipants(locus.participants);
+  }
+
+  onDeltaLocus(locus) {
+    this.updateLocusInfo(locus);
+    this.updateParticipants(locus.participants, true);
+  }
+
+  updateLocusInfo(locus) {
+    this.updateControls(locus.controls);
+    this.updateConversationUrl(locus.conversationUrl);
+    this.updateCreated(locus.created);
+    this.updateFullState(locus.state);
+    this.updateHostInfo(locus.host);
+    this.updateMeetingInfo(locus.info);
+    this.updateMediaShares(locus.mediaShares);
+    this.updateParticipantsUrl(locus.participantsUrl);
+    this.updateReplace(locus.replace);
+    this.updateSelf(locus.self);
+    this.updateLocusUrl(locus.url);
+    this.updateaclUrl(locus.aclUrl);
+    this.updateBasequence(locus.baseSequence);
+    this.updateSequence(locus.sequence);
+    this.updateMemberShip(locus.membership);
+    this.updateIdentifiers(locus.identities);
+  }
+
+  addedOrRemovedParticipant() {
+    return {
+      addedParticipants: [],
+      removedParticipants: [],
+      updatedParticipants: []
+    };
+  }
+
+  updateParticipants(participants, deltaParticpantFlag) {
```

Comment:
can we split this function up?

Reply:
We can do that in the next PR it need more changes to it